client/question/domain/ErrorTracebackObjectFactory.js

```diff
@@ -91,7 +91,7 @@ tie.factory('ErrorTracebackObjectFactory', [
         !this._tracebackCoordinates) {
         return this._errorMessage;
       }
-      return this._errorMessage + ' on line ' + this._getFirstTracebackLine();
+      return this._errorMessage;
```

Comment:
Fixed in upcoming commit. Re: docstring. I'm hoping the changes elsewhere to the codebase should eliminate the expectation that a number will be returned. I wonder if calling that out here would cause some confusion.

Reply:
I think it's fine to update the docstring, because Python typically returns a line number with the error, so the behaviour here is a bit atypical relative to that.

Reply:
ACK Fixed in upcoming commit.